sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -139,7 +139,7 @@ def read(
         :raises ~azure.cosmos.exceptions.CosmosHttpResponseError: Raised if the container couldn't be retrieved.
             This includes if the container does not exist.
         :returns: Dict representing the retrieved container.
-        :rtype: dict[str, Any]
+        :rtype: Dict[str, Any]
```

Comment:
This change is not necessary in a docstring, actually it could even break the link to the Python doc. Docstrings are not using Typing syntax.